auth/service/service/service.go

```diff
@@ -46,14 +49,23 @@ func New() *Service {
 }
 
 func (s *Service) Run() error {
-	errs := make(chan error, 0)
+	errs := make(chan error)
 	go func() {
-		errs <- s.userEventsHandler.Run(context.Background())
+		errs <- s.userEventsHandler.Run()
 	}()
 	go func() {
 		errs <- s.Service.Run()
 	}()
 
+	go func() {
```

Comment:
I think it's best if the definition of this function is defined in `service/service/service.go` or in the runner so this code can be shared. You could define a function that takes a channel as parameter and then just invoke it in each service with `go s.listenForSignals(doneChan)`. I think it's better if you don't use the errors channel - a `doneChan := make(chan struct{})` will be more appropriate.